### DISCUSSION TITLE
Is there an anti-raid bot?

### DISCUSSION THREAD

**Discussion by ghost** [stoatchat/discussions#422] [Q&A] [answered]
Bonjour créer un bot antiraid

**Comment by Rexogamer** [comment, marked as answer]
*Translated with Apple Translate:* "Hello, create an anti-raid bot"

[AutoMod](https://automod.me) has a variety of moderation tools/features, including automatic spam detection (currently enabled by running `/botctl spam_detection on`) and customisable (albeit basic) anti-spam functionality that can be configured via the web UI. While it's not officially first-party, it's maintained by @janderedev (a member of our development team) and is used/trusted in several first-party servers, including the Revolt Lounge. Hope that helps!

**Comment by FillingTheVoidOnYT** [comment]
automod
